sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -557,7 +604,10 @@ Page<IncomingRelationship> NextPageFunc(string nextLink, int? pageSizeHint)
         /// <param name="relationshipId">The Id of the relationship to retrieve.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The application/json relationship corresponding to the provided relationshipId and the http response.</returns>
-        /// <remarks>This returned application/json string can always be serialized into an instance of <see cref="Azure.DigitalTwins.Core.Serialization.BasicRelationship"/> or into an extension of that type.</remarks>
+        /// <exception cref="RequestFailedException">
+        /// The exception that captures the errors from the service. Check the <see cref="RequestFailedException.ErrorCode"/> and <see cref="RequestFailedException.Status"/> properties for more details.
+        /// </exception>
+        /// <remarks>This returned application/json string can always be serialized into an instance of <see cref="Serialization.BasicRelationship"/> or into an extension of that type.</remarks>
```

Comment:
Still not removed

Reply:
I had not seen your other reply. I thought the rebase should remove all of this. Will remove this.